app/common/helpers/validators/uuid-validate.js

```diff
@@ -0,0 +1,18 @@
+import isUuidValid from './uuid';
+
+/**
+ * We need this wrapper because:
+ * 1. Redux-form has strange logic, we need to return undefined if value is valid.
+ * 2. We need to rule error's message.
+ *
+ * @link https://redux-form.com/6.4.3/examples/fieldlevelvalidation/
+ *
+ * @param value string Value to be validated.
+ *
+ * @returns {(undefined|string)} Undefined for ok, string for errors.
+ */
+const uuidValidate = value => {
+  return isUuidValid(value) ? undefined : 'Некоректний формат';
+};
+
```

Comment:
```
const uuidValidate = value => isUuidValid(value) ? undefined : 'Некоректний формат';
```

Reply:
With our new code style (prettier) it will be:
```
const uuidValidate = value =>
  isUuidValid(value) ? undefined : 'Некоректний формат';
```

So I decided to write with return for more readability.